src/Application.php

```diff
@@ -35,15 +35,16 @@ class Application extends BaseApplication
      */
     public function middleware($middleware)
     {
-        // Catch any exceptions in the lower layers,
-        // and make an error page/response
-        $middleware->push(new ErrorHandlerMiddleware());
+        $middleware
+            // Catch any exceptions in the lower layers,
+            // and make an error page/response
+            ->push(new ErrorHandlerMiddleware();
```

Comment:
Since there was a `tests` folder with fixture and test cases I assumed there were valid tests for the files included in the skeleton app. I see now that to even make the testsuite run will require changes in `tests/bootstrap.php`.  Also besides the `PagesControllerTest.php` and `ApplicationTest.php` (which I have actually added in wrong directory) I really don't see the point of all the other files.